libr/core/core.c

```diff
@@ -524,7 +532,15 @@ static ut64 num_callback(RNum *userptr, const char *str, int *ok) {
 			return 0LL; // maybe // return UT64_MAX;
 		case '?': return core->num->value;
 		case '$': return core->offset;
-		case 'o': return r_io_section_vaddr_to_maddr_try (core->io, core->offset);
+		case 'o': 
+			{
+				SdbList *secs;
```

Comment:
repeated pattern several times. maybe it was better with r_io_section_vaddr_to_maddr_try